### PR TITLE
docs: use tabs to show different installation types

### LINKED ISSUE #3133: docs: use tabs to show different installation types
*Description*:
>Describe the issue.

Instead of highlighting in different pages, use tabs to show different ways to install EG
* helm
* kubectl
* egctl

https://gateway.envoyproxy.io/latest/install/

[optional *Relevant Links*:]
>Any extra documentation required to understand the issue.

### PR DESCRIPTION
**Pull request by Manoramsharma**
**What type of PR is this?**
Docs feature enhancement

**What this PR does / why we need it**:

This PR is made to enhance the docs, making it more presentable using tabpane installation.

**Which issue(s) this PR fixes**:

Fixes #3133

*Before*
Screenshot 2024-07-21 at 11 29 00 PM

*After*
Screenshot 2024-07-21 at 11 29 45 PM
Screenshot 2024-07-21 at 11 30 04 PM

### CODE REVIEW

**Comment by zirain** [comment]
image
this looks not right

**Comment by Manoramsharma** [comment]
> image this looks not right

Yes, this was because of the `alert` feature used under the the tabpane

> image this looks not right

Resolved it 👍

File: site/content/en/docs/install/install.md [hunk 1]
```diff
@@ -142,3 +146,113 @@ These are the ports used by Envoy Gateway and the managed Envoy Proxy.
...
+
+{{% /tab %}}
+{{% tab header="Install with Kubernetes YAML" %}}
+In this guide, we'll walk you through installing Envoy Gateway in your Kubernetes cluster.
+
+The manual install process does not allow for as much control over configuration
+as the [Helm install method](./install-helm), so if you need more control over your Envoy Gateway
+installation, it is recommended that you use helm.
+
+## Before you begin
```
Comment (zirain) [inline site/content/en/docs/install/install.md:158]: I think we can move this paragraph out of tabpane

Comment (Manoramsharma) [reply site/content/en/docs/install/install.md:158]: Hi @zirain I tried to move this out of tabpane but it became static and was not able to switch with the switching tabs. Should I attach a preview of it after making it out of tabpane?

**Comment by Manoramsharma** [comment]
Screenshot 2024-07-22 at 3 06 21 PM

See Now this is not changing with the changing tabs?

File: site/content/en/docs/install/install.md (hunk 1 above)
Comment (zirain) [reply site/content/en/docs/install/install.md:158]: `Version Compatibility Matrix` part should be moved out of tabpane.

**Comment by zirain** [comment]
I'm sorry, but you should not direct edit content `/docs`, it alias to current version.
More info can be found [here](https://github.com/envoyproxy/gateway/pull/3819).

**Pull request closed without merging**